cmd/email-provider/server/operations.go

```diff
@@ -16,7 +20,69 @@ func NewServer() *Server {
 }
 
 func (server *Server) SendMessage(ctx context.Context, message *email_provider.SendEmailRequest) (*email_provider.SendEmailResponse, error) {
-	// TODO: send an email to someone here
+	m := mail.NewV3Mail()
+	address := os.Getenv("EMAIL_FROM_ADDRESS")
+	name := os.Getenv("EMAIL_FROM_NAME")
```

Comment:
not just for those 2 env variables, but for all `os.Getenv(...)` calls in this function: what happens if one variable is not defined? There should be some error handling for that.

Reply:
Les variables doivent tous être définis sinon le courriel ne s'envoie pas : )

Reply:
La struct Server pourrait être utilisé pour conserver ca plutot que de lire de env à chaques fois, aussi ca permet de valider que c'est bien configuré dans `NewServer()` et non au moment ou tu va envoyer un email

Reply:
Oui, mais le courriel n'a rien avoir avec la struct Server...

Reply:
oups 😂